blocks/melody/ff_container/ff_container-scrollable-list/ff_container-scrollable-list.less

```diff
@@ -1,12 +1,12 @@
 .ff_container-scrollable-list {
-    width: 100%;
-    border-collapse: collapse;
-    border:2px solid @ff_colour_control_border;
-    height: 585px;
-
-    display: flex;
+	display: flex;
+	box-sizing: border-box;
+	width: 100%;
+	border-collapse: collapse;
+	border:2px solid @ff_colour_control_border;
+	height: 585px;
+	flex-direction: column;
     align-items: stretch;
-    box-sizing: border-box;
     &__item {
 
     }
```

Comment:
Something here seems to have broken things, look at:
http://localhost:3000/blocks/core/lib_test/lib_test-task-responses-screen/lib_test-task-responses-screen.html